supertokens_python/recipe/thirdpartyemailpassword/utils.py

```diff
@@ -33,14 +39,16 @@
 
 
 class InputEmailVerificationConfig:
+    # TODO: FIXME
+    # FIXME: Why is this duplicate of emailpassword.utils.InputEmailVerificationConfig?
     def __init__(self,
-                 get_email_verification_url: Union[Callable[[
-                     User, Any], Awaitable[str]], None] = None,
-                 create_and_send_custom_email: Union[Callable[[
-                     User, str, Any], Awaitable[None]], None] = None
+                 get_email_verification_url: Union[Callable[[User, Any], Awaitable[str]], None] = None,
+                 create_and_send_custom_email: Union[Callable[[User, str, Any], Awaitable[None]], None] = None
                  ):
         self.get_email_verification_url = get_email_verification_url
         self.create_and_send_custom_email = create_and_send_custom_email
+        if create_and_send_custom_email:
+            warn("create_and_send_custom_email is depricated. Please use email delivery config instead")
```

Comment:
Wrong way of deprecating

Reply:
That's all we can do in Python. Using our util func `deprecated_warn`.